hack/dind-cluster.sh

```diff
@@ -80,7 +80,7 @@ function start() {
     # dockershim is default and doesn't need an endpoint path
     runtime_endpoint=
   elif [[ "${container_runtime}" = "crio" ]]; then
-    runtime_endpoint="/var/run/crio.sock"
+    runtime_endpoint="unix:///var/run/crio/crio.sock"
```

Comment:
this gets rid of a warning at startup but doesn't actually change any behavior, right?